viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1685,7 +1694,9 @@ export class Cognite3DViewer {
     const start = Date.now();
 
     this._events.beforeSceneRendered.fire({ frameNumber, renderer: this.renderer, camera });
-
+    this._sceneHandler.customObjects.forEach(customObject => {
+      customObject.beforeRender(camera);
+    });
```

Comment:
Should the `beforeRender`-funcion of each `customObject` be added to the `this._events.beforeSceneRendered` trigger instead of manually getting called here?

Reply:
It was like that it worked before on the architeture side. But the loop was too compicated